spec/models/location_spec.rb

```diff
@@ -68,9 +62,14 @@
           "1" => ActionController::Parameters.new(address: 'wrong')
         ).permit!
       )
+    }
 
-      expect(location.save).to be_falsey
+    it 'is invalid' do
```

Comment:
Needs to be more descriptive for instance "does not create the location" or "does not pass validation"